plugin/src/main/java/fr/mrmicky/worldeditselectionvisualizer/placeholders/PlaceholderManager.java

```diff
@@ -0,0 +1,39 @@
+package fr.mrmicky.worldeditselectionvisualizer.placeholders;
+
+import fr.mrmicky.worldeditselectionvisualizer.WorldEditSelectionVisualizer;
+import me.clip.placeholderapi.PlaceholderAPI;
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.scheduler.BukkitRunnable;
+
+public class PlaceholderManager {
+    private final WorldEditSelectionVisualizer plugin;
+    private PlaceholderExpansion placeholders = null;
+
+    public PlaceholderManager(WorldEditSelectionVisualizer plugin) {
+        this.plugin = plugin;
+    }
+
+    public void hook() {
+        placeholders = new PlaceholderHandler(plugin);
+
+        new BukkitRunnable(){
+            @Override
+            public void run() {
+                placeholders.register();
+            }
+        }.runTask(plugin);
+    }
+
+    @SuppressWarnings("all")
```

Comment:
A `@SuppressWarnings` with only the warning to suppress would be better than removing all warnings in the method
```suggestion
    @SuppressWarnings("deprecation")
```